presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -66,6 +66,7 @@
 import com.facebook.presto.sql.planner.iterative.rule.RemoveTrivialFilters;
```

Comment:
please sort commits before posting a PR, I expected this commit to be the last one, not the first.